src/field/username.js

```diff
@@ -55,5 +55,5 @@ export function setUsername(m, str, usernameStyle = 'username', validateUsername
 }
 
 export function usernameLooksLikeEmail(str) {
-  return str.indexOf('@') > -1;
+  return str.indexOf('@') > -1 && str.indexOf('.') > -1;
```

Comment:
Wonder if we should check the '.' comes after the @ here?

```javascript
return str.indexOf('.') > str.indexOf('@') && str.indexOf('@') > -1;
```

Reply:
this would return false for `test.email@t.com`